src/Exploded.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ElegantBro\Arrayee;
+
+use ElegantBro\Interfaces\Arrayee;
+use ElegantBro\Interfaces\Stringify;
+use Exception;
+
+class Exploded implements Arrayee
+{
+    /**
+     * @var Stringify glue
+     */
+    private $glue;
+
+    /**
+     * @var Stringify stringify to explode
+     */
+    private $stringify;
+
+    public static function byComma(Stringify $stringify): Exploded
+    {
+        return new static(new \ElegantBro\Stringify\Just(","), $stringify);
```

Comment:
Looks like we need to have `Just` implementations for every type in `Interfaces` library .  You can do without `byComma` constructor at this time and do not import `Stringify\Just`

Reply:
Done!